.gitignore

```diff
@@ -24,3 +24,6 @@ profiles.clj
 resources/contracts
 node_modules
 .DS_Store
+src/java
+yarn.lock
+env/dev/resources/config.edn
```

Comment:
When changing that config file, it hangs in git status causing a chance to commit my credentials eventually. The better approach, I believe, would be to provide an sample of that file giving instructions in readme.

Reply:
Agree with the sample file! And we could even separate normal config from sensitive config by using two config files, maybe `config.edn` and `config-private.edn`, so that we are more aware of that.

Reply:
Or how about instead of having a sample having a `config-default.edn as the sample, which could be imported alongside `config-private.edn`, this one overwriting the custom values from the other one, and this one not being versioned?